### PR TITLE
milestone 4: Game instructions

### PR DESCRIPTION
**Pull request by evansinho**
### On this milestone:
- updated the readme file with instruction on how to play the game

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,69 @@
 # Project Name
```
Comment (uryelah) [inline README.md]: - [x] Please customize the whole readme file, including the 'Project name'.

File: README.md [hunk 2]
```diff
@@ -1,73 +1,69 @@
...
+- Ruby,
+- Vs.code,
+- Git and Github
 
-[Live Demo Link](https://livedemo.com)
+## Getting Started
 
+- clone this repository `https://github.com/rajkumardas2701/Tic-Tac-Toe`
+- cd to the folder `Tic-Tac-Toe`
+- To run the application use this command `ruby bin/main.rb`
```
Comment (uryelah) [inline README.md:17]: And after this?
- [x] Explicitly tell the user how your game interface works and the commands they will have to input. Be explicit about it being a terminal/command prompt game.

- [x] [OPTIONAL] Add a screenshot or code snippet showing the game UI

File: README.md [hunk 3]
```diff
@@ -1,73 +1,69 @@
...
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- Odin Project
+- Ferguson Iyara
 
 ## 📝 License
```
Comment (uryelah) [inline README.md:68]: - [x] If you won't include the license type it's better to just exclude the license header.

**Review by uryelah — CHANGES_REQUESTED** [review]
Hello **Evanson Igiri & Rajkumar Das Das**, I'm Sarah, your reviewer for this PR 👩‍🚀  

Good job getting started wth the Readme file!

There are still a couple of requirements missing, check the comments below 🦾 

--------

## Changes Required 🏗️ 🐛 💡

**Comment by uryelah** [comment]
**Kindly fix the pointed out issues before requesting the next code review**

---------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

File: README.md (hunk 3 above)
Comment (evansinho) [reply README.md:68]: A licence type is present

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hi @evansinho, @rajkumardas2701. You have done well to include the game rules and instructions:+1:**

The previous reviewer pointed to your license section. Even though you have mentioned MIT license, it doesn't link to any license document. You can use this [link](https://license-generator.intm.org/) to generate an MIT license text and add it to this project OR Use the answer provided [here](https://stackoverflow.com/a/52466628) to add it directly on GitHub.
Kindly do this to make your README perfectly professional.

**Cheers:stars:**

### Status: Changes Requested.

**Comment by rajkumardas2701** [comment]
Hi @mosesogwo and Team, 

Updated MIT License in Readme as suggested.


Thanks,
Rajkumar Das

**Review by maelfosso — APPROVED** [review]
# Congratulations :clap: 

You can merge :smile: 
Good continuation :handshake: 

Mael FOSSO

**Pull request merged**